CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# x.x.x
+
+- feat: add improved example project
```

Comment:
can we remove this changelog part? we can address that on #53

Reply:
Good catch. Will do!